engine/runtime-instrument-common/src/main/scala/org/enso/interpreter/instrument/execution/JobExecutionEngine.scala

```diff
@@ -131,6 +131,12 @@ final class JobExecutionEngine(
               }
               logger.log(Level.WARNING, sb.toString())
               runningJob.future.cancel(runningJob.job.mayInterruptIfRunning)
+            case _: CancellationException =>
```

Comment:
Is this related to native-image?

Reply:
Indirectly. @jdunkerley reported it while using NI but I suppose one could get this exception with JVM runner as well.